packages/archive/store-engine-sqleet/src/SQLeetEngine.test.ts

```diff
@@ -148,7 +148,6 @@ describe('SQLeetEngine', () => {
       try {
         const entity = {'name\'"`': 'Otto'};
         await engine.create('users', '1', entity);
-        fail();
```

Comment:
Why?!

Reply:
Good catch. Fail doesn't exist in jest, but I need to throw an error here